sql/moz-fx-data-shared-prod/fenix_derived/retention_v1/backfill.yaml

```diff
@@ -1,4 +1,4 @@
-2024-05-30:
+2024-05-31:
   start_date: 2021-01-01
   end_date: 2024-05-31
```

Comment:
Is this table in prod/scheduled already? Shouldn't there be some data on or around `2024-05-31`? Normally this is why `end_date`'s are in the recent past.